src/binding.cc

```diff
@@ -22,6 +30,8 @@ void InitModule(Handle<Object> target) {
   Nan::Set(target,
     Nan::New<String>("version").ToLocalChecked(), 
     Nan::New<String>(TF_Version()).ToLocalChecked());
+  target->Set(Nan::New("GetAllOpList").ToLocalChecked(),
+               Nan::New<v8::FunctionTemplate>(GetAllOpList)->GetFunction());    
```

Comment:
What I mean is to move this method on `Graph`, note that that is still a static method. Utils is not suitable for this case IMO.